modules/drivers/bigquery/src/metabase/driver/bigquery.clj

```diff
@@ -406,8 +406,9 @@
 ;;; |                                Other Driver / SQLDriver Method Implementations                                 |
 ;;; +----------------------------------------------------------------------------------------------------------------+
 
-(defmethod driver/date-interval :bigquery [driver unit amount]
-  (sql.qp/->honeysql driver (du/relative-date unit amount)))
+(defmethod driver/date-add :bigquery
+  [driver dt amount unit]
```

Comment:
I would vote for using [`_` instead of `driver` here](https://github.com/bbatsov/clojure-style-guide#underscore-for-unused-bindings) so it's clear it's not being used